### ISSUE TITLE
Lighthouse Audit - Accessibility Enhancements

### ISSUE THREAD

**Issue by abiddiscombe** [abiddiscombe/notedeck#53]
A Lighthouse audit has revealed the following Accessibility and SEO issues:

**"Buttons do not have an accessible name"** \
All buttons within the header must use the `aria-label` attribute.
Modal close buttons (which use the `X` icon) must use the `aria-label` attribute.

**"Background and foreground colors do not have a sufficient contrast ratio"** \
The NoteDeck version tag must use a darker shade of the primary color.
The "Restore from Backup" file selector does not have a suitable contrast ratio.

**Investigate opportunities to remove "Splash Screen" on mobile devices.**
There's no need to initialise IndexedDB or fetch release data from GitHub's API when NoteDeck is viewed on mobile devices, since this is a desktop-only application and mobile users will see the "App Unsupported" message.
